# Eisa Mahyari, Ph.D.

## Contact 
[Google Scholar Citations](https://scholar.google.com/citations?user=vp6qss8AAAAJ&hl=en), [Research Gate](https://www.researchgate.net/profile/Eisa_Mahyari), [Github](https://eisascience.github.io/), [Twitter](https://twitter.com/eisamahyari)

## Education
B.S. - Biology and Chemistry double major. Portland State University (PSU), Portland, OR. (2002 – 2006)

M.S. - Biochemistry and Molecular Biology. Oregon Health and Science University (OHSU), Div. Environmental and Biomolecular Systems, Portland, OR. “A six-compartment Physiologically Based Pharmacokinetic (PBPK) Markov-chain Monte Carlo Model to simulate and quantify nicotine metabolism in human populations.” (2006 – 2008).

Ph.D. - Bioinformatics and Computational Biology, OHSU, Department of Medical Informatics and Clinical Epidemiology (DMICE), Division of Bioinformatics and Computational Biology (BCB), Portland, OR. “Robust and reproducible classification of rare cellular subsets/signatures (RCS) in single-cell technologies within a transfer learning framework.” (doi:10.6083/NX9EZJ) Committee members: Shannon K. McWeeney Ph.D., David M. Lewinsohn MD./Ph.D., Michael Mooney Ph.D., Christina Zheng Ph.D., and Evan Lind Ph.D. (2013 – 2018).

## About Me
I am computational biologist focused and interested on solving problems related to immunology in the context of infectious disease and oncology. I have recieved my Ph.D. in [Bioinformatics and Computational Biology (BCB)](https://www.ohsu.edu/xd/education/schools/school-of-medicine/departments/clinical-departments/dmice/educational-programs/dmice-programs/computational-biology.cfm) from [Oregon Health and Science Unitversity (OHSU)](https://www.ohsu.edu/) in June 2018. I am profoundly fascinated by the immune system and the mechanisms of immunity to develop treatments and prophylactics, specifically within a precision-medicine framework.

My prior training was in Biology and Chemistry (BS 2006) and Biochemistry and Molecular Biology (MS 2008). From 2008 to 2012 I was a group member in Dr. Louis Picker's Lab, where I focused on immunological studies relating to HIV Vaccines. Our work culminated in a critical examination of vaccine-mediated correlates of protection in Rhesus Macaques [(Nature Medicine 2012)](https://www.ncbi.nlm.nih.gov/pubmed/22961108). This experience seeded the interest and desire to contribute to multi-disciplinary translational biomedical research which propelled me to apply and be accepted as a PhD student in the Division of BCB, in the Department of Medical Informatics and Clinical Epidemiology (DMICE).

As a PhD candidate, I worked under the guidance and mentorship of Drs. David M. Lewinsohn, Marielle C. Gold and Shannon K. McWeeney. I contributed to projects and grants studying a population of tuberculosis (TB)-recognizing T-cells known as mucosal associated invariant T (MAIT) cells. A key idea of these studies was that MAIT cells may be harnessed as unconventional TB vaccine targets.

For my Ph.D. thesis [(robust and reproducible classification of rare cellular subsets/signatures (RCS) in single-cell technologies within a transfer learning framework)](https://doi.org/10.6083/nx9ezj), I focused on strategies for computational immune phenotyping using single-cell technologies (i.e., flow cytometry, CyTOF, and RNA-Seq). Specifically, I developed a machine learning framework [(the RTL framework, available on Github)](https://github.com/eisascience/RTL) to classify rare cellular subsets in single-cell data because classification of rare events from high-dimensional data is a difficult and highly variable task for humans and machines. Such robust and reproducible methods are a major pillar for accurate predictions of treatment/prophylactic effectiveness and prognostics within a precision-medicine framework. For the near future, I am interesting in continuing to develop robust computational methods to integrate and processes biomedical data as well as addressing clinical hypotheses. Long term, I am interested integrating the mechanisms of immunity and related data to better stratify patients and predicting adverse/positive outcomes.

## Publications (h-index = 1, citations = 133)
Yoshinori Fukazawa, Haesun Park, Mark J. Cameron, Francois Lefebvre, Richard Lum, Noel Coombes, __Eisa Mahyari__, Shoko I. Hagen, Jin Young Bae, Marcelo Delos Reyes, Tonya Swanson, Alfred W. Legasse, Andrew Sylwester, Scott G. Hansen, Andrew T. Smith, Petra Stafova, Rebecca Shoemaker, Yuan Li, Kelli Oswald, Michael K. Axthelm, Adrian McDermott, Guido Ferrari, David C. Montefiori, Paul T. Edlefsen, Michael Piatak, Jeffrey D. Lifson, Rafick P. Sékaly, and Louis J. Picker.   “Lymph node T cell responses predict the efficacy of live attenuated SIV vaccines.” Nature Medicine, 18(11):1673–1681, 2012. 

## Oral & Poster Presentations
Winter 2014   “Latent Dirichlet Allocation and Application in Biology”, Bioinformatics Discussion for Students and Postdocs (BioDSP), OHSU.
Spring 2015   “A Reinforcement Learning Model for Solving the Protein Folding Problem”, BioDSP, OHSU
Spring 2015   “Identification of novel MAIT subtypes with non-linear dimensionality reduction”, Research Week poster, OHSU.
Spring 2018   “Robust and reproducible classification of rare cellular subsets/signatures (RCS) in single-cell technologies within a transfer learning framework”, Research Week talk, OHSU.

## Teaching Experience
Spring 2015	Class TA for Statistical methods with course professor: Shannon K. McWeeney Ph.D.
Spring 2016	Class TA and Mentored teaching, "Markov Models, Hidden Markov Models, and Markov Chain Mote Carlo Modeling" for Statistical methods with course professor: Shannon K. McWeeney Ph.D.
Spring 2017	Class TA & Mentored teaching, "Public and social-media engagement and the developing ‘avatar’ of an academic professional" for Research in Bioinformatics and Computational Biology with course professor: Eilis Boudreau M.D./Ph.D.

## Research Products
“TCR Enrichment Analysis (TEA)” Visualization interface to evaluate differential enrichment of tissue-specific TCR clonotype. 

## Awards and Honors
2013 – 2014     Knight Cancer Institute, pre-doctoral funding, OHSU. \\
2013 – 2014     Molecular Microbiology and Immunology, pre-doctoral funding, Paul Spellman Ph.D., OHSU.\\
2014 – 2016     Pulmonary/Critical Care Medicine. NIH T32 Grant, David Lewinsohn M.D./Ph.D., OHSU (5T32HL083808-06/07). \\



[Follow @eisamahyari](https://twitter.com/eisamahyari)
