# Eisa Mahyari, Ph.D.

## Contact 
[Google Scholar Citations](https://scholar.google.com/citations?user=vp6qss8AAAAJ&hl=en) | [Research Gate](https://www.researchgate.net/profile/Eisa_Mahyari) | [ORCID](https://orcid.org/0000-0002-5613-3131) | [GitHub](https://eisascience.github.io/) | [Twitter](https://twitter.com/eisamahyari) | [Email](mailto:eisamahyari@gmail.com)

| **CORE SKILLS**              | **Management**                          | **Statistics & Computer Science**            | **Biology**                          |
|------------------------------|-----------------------------------------|----------------------------------------------|--------------------------------------|
|                              | - Project coordination                  | - Machine Learning & AI                      | - Single-cell & spatial OMICs        |
|                              | - Operational & cross-functional leadership | - Fine-tuning of LLM on specialized domains | - Translational research             |
|                              | - Multi-disciplinary collaboration      | - Software engineering                       | - Reproduction & infertility         |
|                              | - Data-driven strategic planning & consultation | - Cloud computing (Slurm, AWS, Google) | - Immunology & infectious diseases   |
|                              | - Budgeting & resource allocation       |                                              | - HIV, MTb, Yellow Fever, Dengue, Zika, CMV, and SARS-CoV-2 |
|                              | - Stakeholder communication             |                                              | - PBPK modeling & pharmacokinetics   |
|                              | - Grant writing                         |                                              |                                      |


## Summary Statement
I am a computational biologist driven by applied machine learning and AI for translational biomedical research. Broadly, I specialize in immunology, infectious diseases, and reproductive biology. I enjoy leading diverse projects, coding, performing statistical analyses, and converting complex biological data into actionable insights alongside principal investigators and cross-disciplinary teams. From a biological perspective, 

As R. Assistant Professor at OHSU/ONPRC, I deploy computational pipelines, develop interactive data visualization platforms, secure and manage research funding through competitive grants, and lead the strategic development of AI/ML solutions for research. My research in this role focuses on integrating high-dimensional multi-omics datasets, performing cross-species analysis, and employing statistical inference and computational omics to support clinical and preclinical initiatives. Furthermore, I actively consult across labs, providing technical training and mentorship to graduate students and guiding their research and career development. 

My career has been defined by consistent advancement and a focus on statistical, machine learning, and data analytic methods relevant to translational and clinical research. I have played a pivotal role in designing, implementing, and analyzing extensive research projects, combining wet lab and in silico methodologies to develop hypotheses and transform data into actionable outcomes. My work upholds rigorous quality control across various omics data types to ensure integrity and reproducibility. Through statistical consultations and analyses, I have contributed significantly to clinical study documentation, scientific publications, presentations, grants, and regulatory filings in academic and industry settings.

Currently, I am interested in leading and developing teams to create software, pipelines, and testing methodologies tailored for analyzing and translating complex biomedical data, supporting impactful translational research by integrating data types, and applying robust statistical methodologies and computational tools. With the advent of large language models (LLMs), such tasks and aims can be rapidly scaled; thus, my research includes fine-tuning, testing, and deploying specialized LLM for biomedical research. 

### Domains of Research Expertise & Interest
- **Immunology**: Focused on adaptive immunity, emphasizing T cell biology and T cell receptors (TCRs) across human, rhesus, and murine models.
- **Sex/Gender Biased Immune function**: Investigating the X chromosome and the vast and complex genes it codes, many of which are related to immune functions. Certain human diseases that alter X-chromosome expression, such as Klinefelter's syndrome, are potential models of investigating these genes. 
- **Reproductive Biology**: Testis biology and genetics of infertility in human, rhesus, and murine models.
- **Infectious Diseases**: Research in HIV/SIV, SARS-CoV-2, ZIKA, Yellow Fever, Dengue, and Mycobacterium tuberculosis.
- **Computational Science**: Statistical inference and machine learning, specializing in matrix/tensor decomposition and transfer learning.
- **Large Language Models (LLMs)**: Applications and methodologies for LLMs in biomedical research.


## Technical skills
- **Bioinformatics**: Proficiency with tools like Bowtie, BLAST, BWA, GATK, SAMtools, STAR, and HTSeq for DNA/RNA alignment, variant calling, and transcript quantification.
- **Cloud computing and HPC**: Experience with high-performance computing (HPC) environments and cloud platforms such as AWS, Google Cloud Platform (GCP), and Azure, utilizing tools like Docker, Conda, and Slurm for scalable data processing and pipeline deployment.
- **Programming languages**: Expertise in R, Python, and Bash scripting for data analysis, automation, and pipeline development.
- **High-throughput data analysis**: Proficient in next-generation sequencing (NGS) and multi-omics data, including RNA-seq, whole-genome sequencing (WGS), whole-exome sequencing (WES), ChIP-seq, ATAC-seq, single-cell, and spatial sequencing.
- **Data visualization**: Skilled in creating publication-quality and interactive data visualizations using ggplot2, Matplotlib, Plotly, and Shiny, along with automated and reproducible reporting in HTML and R Markdown.
- **Data science and machine learning**: Competent in supervised and unsupervised machine learning techniques (e.g., random forests, SVM, clustering, dimensionality reduction) and deep learning frameworks (e.g., TensorFlow, PyTorch) for predictive modeling and data interpretation.
- **Mathematical Modeling**: Sytems Biology and Physiological Based Pharmacokinetic (PBPK) modeling with applied Markov Chain Monte Carlo.
- **Advanced statistical and computational methods**: Proficient in Bayesian inference, survival analysis, mixed-effects modeling, differential expression analysis, dimensionality reduction (PCA, UMAP, t-SNE, SDA), and advanced regularization techniques, enabling complex biological insights from high-dimensional datasets.


## Professional Experience

### 2022 - Present  
**Research Assistant Professor, Mammalian Reproduction**  
_Lab of Donald F. Conrad, Ph.D. - Division of Genetics, ONPRC-OHSU_

- Lead computational research projects to analyze high-dimensional multi-omics data, advancing translational research in testis biology and male infertility.
- Oversee the design, development, and refinement of data pipelines, enabling the seamless integration and analysis of genomic, transcriptomic, and epigenomic data to identify biomarkers and improve understanding of reproductive genetics.
- Mentor and supervise graduate students and lab members, providing training in statistical methodologies, computational tools, and project management to enhance team capability.
- Collaborate cross-functionally with biologists, geneticists, and clinical researchers to translate complex data into actionable insights, driving forward research initiatives with a strategic focus on clinical relevance and innovation.
- Managed collaborative, multi-institutional projects, coordinating resources and ensuring robust quality control in data integrity and analysis.
- Collaborated with cross-functional teams, including R&D, IT, engineering, veterinarians, and clinicians.


### 2018 - 2022  
**Computational Biologist, Molecular Immunopathology - Research Associate**  
_Lab of Benjamin N. Bimber, Ph.D. - Primate Genetics, ONPRC-OHSU_

- Developed hypotheses and transformed complex immunogenomic data into key insights that advance the understanding of immunopathology and identify genomic signatures of pathogen infection, contributing to new therapeutic directions and identification of diagnostic biomarkers.
- Designed and implemented pipelines, algorithms, and machine learning models for multi-omics analysis, especially single-cell transcriptomics, to uncover novel genetic markers and disease mechanisms.
- Delivered comprehensive technical training in computational biology and machine learning to researchers and students, fostering expertise in data analysis, statistical programming, and project-based learning.

### 2022  
**Biostatistics & Computational Biology Consultant**  
_Drug Development & Testing for Nonalcoholic Steatohepatitis (NASH) - CytoDyn, USA_

- Established and executed scalable analysis frameworks for multi-omics clinical data, integrating machine learning and statistical methods to support drug development for NASH.
- Conducted quality control on complex clinical datasets, ensuring data reliability and supporting milestones for clinical-stage programs.
- Partnered with cross-functional teams, translating statistical findings into actionable clinical insights that guided drug development and trial decisions, thereby enhancing trial outcomes.

### 2018 - 2022  
**Computational Biologist, Reproductive Biology - Research Associate**  
_Lab of Donald F. Conrad, Ph.D. - Reproductive Genetics, ONPRC-OHSU_

- Identified genomic and transcriptomic signatures that differentiate clinical profiles in male infertility, utilizing scRNA-seq and integrative omics approaches.
- Designed and implemented scRNA-seq experiments with murine models to investigate specific hypotheses, advancing translational applications in reproductive health.
- Developed and customized bioinformatics algorithms, adapting emerging methodologies to lab-specific needs for data analysis, quality control, and reproducibility.
- Collaborated closely with a multidisciplinary team of scientists, engineers, and clinicians to drive discoveries and innovation in reproductive genetics.

2016 - 2018      __Doctoral Candidate__, DMICE-BCB, OHSU. Developed novel machine learning framework for automated classification of rare cells in single-cell RNA-seq data.
2013 - 2016      __Doctoral Student__, DMICE-BCB, OHSU. 

Research Involvement:

+ __Genome-based Bioinformatics__, Assessment of bioinformatics pipelines identifying genomic structural variants across several sequenced Gibbons. Lucia Carbone Ph.D.

+ __Computational phenotyping in cancer cell lines__, CyTOF-based computational analysis of phenotypic heterogeneity in breast cancer-derived cell in the context of combinatory anti-cancer drug treatments to identify sensitive phenotypic subtypes within several breast cancer cell lines. Paul T. Spellman Ph.D.

+ __Computational immunophenotyping__, CyTOF-based computational immunophenotypic analysis of a unique population of tuberculosis (TB)-recognizing T-cells known as mucosal-associated invariant T (MAIT) cells. David M. Lewinsohn M.D./Ph.D.

2008 - 2012      __Sr. Research Associate__, Louis Picker M.D., Vaccine & Gene Therapy Institute (VGTI), OHSU. 
Contribution: Performed experiments, analyzed flow cytometric data, performed statistical tests and developed reports and presentation materials. 

2005 - 2007      __Research Associate__, Clinical Research Investigative Studies Program (CRISP) and Critical Care Academics Associates Program (CCAAP), OHSU.
Contribution: Acquired clinical data from patient charts for ongoing studies and identified candidates for enrollment into qualifying research in the emergency department and the intensive care units.  


## Personal & Professional Attributes

- **Strategic and Business-Oriented**: Strong strategic thinker with excellent business acumen, aligning initiatives with organizational goals to drive impactful outcomes.
- **Collaborative Leadership**: Proven leader with a collaborative approach, fostering teamwork across multidisciplinary environments.
- **Entrepreneurial Mindset**: Resourceful and adaptable; thrives in fast-paced, dynamic settings and embraces new challenges with innovative solutions.
- **Hands-On and Tactical**: Skilled in both strategic planning and hands-on execution, with a focus on translating high-level goals into actionable results.
- **Results-oriented Problem Solver**: Dedicated to achieving measurable outcomes through effective problem-solving and a focus on continuous improvement.
- **Mission-Driven**: Passionate about advancing science and health through meaningful, mission-aligned contributions.
- **Integrity and Trustworthiness**: Upholds the highest standards of integrity, respect, and reliability in professional and personal interactions.


## Education
- **Post-Ph.D.** – Mammalian Reproduction, Oregon Health & Science University (OHSU), Oregon National Primate Research Center (ONPRC), Division of Genetics  
  *Advisor: Prof. Don F. Conrad*
  
- **Ph.D.** – Bioinformatics and Computational Biology, OHSU, Department of Medical Informatics and Clinical Epidemiology (DMICE), Division of Bioinformatics and Computational Biology (BCB), Portland, OR  
  *Dissertation*: "Robust and reproducible classification of rare cellular subsets/signatures (RCS) in single-cell technologies within a transfer learning framework." [(doi:10.6083/NX9EZJ)](https://doi.org/10.6083/nx9ezj)  
  *Committee*: Shannon K. McWeeney, Ph.D., David M. Lewinsohn, M.D./Ph.D., Michael Mooney, Ph.D., Christina Zheng, Ph.D., Evan Lind, Ph.D.  
  *(2013 – 2018)*

- **M.S.** – Biochemistry and Molecular Biology, OHSU, Division of Environmental and Biomolecular Systems, Portland, OR  
  *Thesis*: "A six-compartment Physiologically Based Pharmacokinetic (PBPK) Markov-chain Monte Carlo Model to simulate and quantify nicotine metabolism in human populations."  
  *(2006 – 2008)*

- **B.S.** – Biology and Chemistry, Portland State University (PSU), Portland, OR  
  *(2002 – 2006)*

## Publications by year

+  __Mahyari, Eisa__, et al. "Rhesus Immune Reference Atlas (RIRA)" In review (2024).

+  __Mahyari, Eisa__, et al. "HLA-B27, AS, and AAU in a clinical cohort" In review (2024).

+ Mariya B. Shapiro, Tracy Ordonez, Shilpi Pandey, __Eisa Mahyari__, Kosiso Onwuzu, Jason Reed, Heather Sidener et al. "Immune perturbation following SHIV infection is greater in newborn macaques than in infants." JCI insight (2024).

+ Sujittra Khampang, Chanchao Lorthongpanich, Chuti Laowtammathron, Phatchanat Klaihmon, Sukanya Meesa, Wichuda Suksomboon, Nittaya Jiamvoraphong, __Eisa Mahyari__ et al. "The dynamic expression of YAP is essential for the development of male germ cells derived from human embryonic stem cells." Scientific Reports 14, no. 1 (2024): 15732.

+ Lillepea, Kristiina, Anna-Grete Juchnewitsch, Laura Kasak, Anu Valkna, Avirup Dutta, Kristjan Pomm, Olev Poolamets, __Eisa Mahyari__ et al. "Toward clinical exomes in diagnostics and management of male infertility." The American Journal of Human Genetics 111, no. 5 (2024): 877-895.

+ Juchnewitsch, Anna-Grete, Kristjan Pomm, Avirup Dutta, Erik Tamp, Anu Valkna, Kristiina Lillepea, __Eisa Mahyari__ et al. "Undiagnosed RASopathies in infertile men." Frontiers in endocrinology 15 (2024): 1312357.

+ __Mahyari, Eisa__, Katinka A. Vigh‐Conrad, Clément Daube, Ana C. Lima, Jingtao Guo, Douglas T. Carrell, James M. Hotaling, Kenneth I. Aston, and Donald F. Conrad. "The human infertility single‐cell testis atlas (HISTA): an interactive molecular scRNA‐Seq reference of the human testis." Andrology (2024).

+ Liina Nagirnaja, Alexandra M. Lopes, Wu-Lin Charng, Brian Miller, Rytis Stakaitis, Ieva Golubickaite, Alexandra Stendahl, Tianpengcheng Luan, Corinna Friedrich, __Eisa Mahyari__, [...], Kristian Almstrup, Kenneth I Aston, Donald F Conrad "Diverse monogenic subforms of human spermatogenic failure", Nature Communications, 13, 1, (2022). [link](https://doi.org/10.1038/s41467-022-35661-z)

+ Yoshinori Fukazawa, Jeremy Smedley, Bryan Randall, __Eisa Mahyari__, Carly Starke, Richard Lum, Manuel Medina, Benjamin Varco-Merth, Kathleen Busman-Sahay, Miranda Fisher, "Mesenteric lymph nodes are sites of increased SIV RNA expression in SIV-infected rhesus macaques on antiretroviral therapy", Journal of Medical Primatology, 51, 5, 320-320, (2022)

+ GW McElfresh, Gregory Boggy, __Eisa Mahyari__, Michael Stutz, Abigail Ventura, Scott Hansen, Louis Picker, Benjamin Bimber, "Single cell transcriptomic profiling of early tuberculosis infection and granuloma formation in rhesus macaque", Journal of Medical Primatology, 51, 5, 313-314, (2022)

+ __Eisa Mahyari__, Abigail Ventura, Eric McDonald, Gregory Boggy, GW McElfresh, Scott Hansen, Louis Picker, Benjamin Bimber, "Rhesus Immunome Reference Atlas (RIRA): A multi-tissue single-cell landscape of immune cells", Journal of Medical Primatology, 51, 5, 296-296, (2022)

+ Gregory J Boggy, GW McElfresh, __Eisa Mahyari__, Abigail B Ventura, Scott G Hansen, Louis J Picker, Benjamin N Bimber "BFF and cellhashR: analysis tools for accurate demultiplexing of cell hashing data." Bioinformatics (2022). [link](https://academic.oup.com/bioinformatics/article-abstract/38/10/2791/6585402)

+ __Eisa Mahyari__, Jingtao Guo, Ana C. Lima, Daniel P. Lewinsohn, Alexandra M. Stendahl, Katinka A. Vigh-Conrad, Xichen Nie, Liina Nagirnaja, Nicole B. Rockweiler, Douglas T.Carrell, James M.Hotaling, Kenneth I.Aston, Donald F.Conrad  "Comparative single-cell analysis of biopsies clarifies pathogenic mechanisms in Klinefelter syndrome." The American Journal of Human Genetics 108, no. 10 (2021): 1924-1945. [link](https://www.sciencedirect.com/science/article/abs/pii/S0002929721003372)

+ N Haigwood, M Shapiro, T Cheever, S Pandey, __E Mahyari__, K Onwuzu, J Reed, H Sidener, J Smedley, L Colgin, A Lewis, A Johnson, B Bimber, J Sacha, A Hessell "Immune perturbation is more profound in newborn than in infant macaques during acute SHIV infection." Journal of the International AIDS Society (2021). [link](https://go.gale.com/ps/i.do?id=GALE%7CA656303588&sid=googleScholar&v=2.1&it=r&linkaccess=abs&issn=17582652&p=AONE&sw=w&userGroupName=oregon_oweb)

+ Bruce K. Patterson, Harish Seethamraju, Kush Dhody, Michael J. Corley, Kazem Kazempour, Jay Lalezari, Alina PS Pang, __Eisa Mahyari__ et al. "CCR5 inhibition in critical COVID-19 patients decreases inflammatory cytokines, increases CD8 T-cells, and decreases SARS-CoV2 RNA in plasma by day 14." International Journal of Infectious Diseases 103 (2021): 25-32.[Link](https://www.sciencedirect.com/science/article/pii/S1201971220323055)

+ Husam Taher, __Eisa Mahyari__, Craig Kreklywich, Luke S Uebelhoer, Matthew R McArdle, Matilda J Moström, Amruta Bhusari, Michael Nekorchuk, Travis Whitmer, Elizabeth A Scheef, Lesli M Sprehe, Dawn Roberts, Colette M Hughes, Kerianne A Jackson, Andrea N Selseth, Abigail B Ventura, Yujuan Yue, Kimberli A Schmidt, Jason Shao, Paul T Edlefsen, Jeremy Smedley, Richard J Stanton, Michael K Axthelm, Jacob D Estes, Scott G Hansen, Amitinder Kaur, Peter A Barry, Benjamin N Bimber, Louis J Picker, Daniel N Streblow, Klaus Früh, Daniel Malouli. "In vitro and in vivo characterization of a recombinant rhesus cytomegalovirus containing a complete genome." PLoS pathogens, 2020. [Link](https://doi.org/10.1371/journal.ppat.1008666).

+ Helen L Wu, Whitney C Weber, Christine Shriver‐Munsch, Tonya Swanson, Mina Northrup, Heidi Price, Kimberly Armantrout, Mitchell Robertson‐LeVay, Jason S Reed, Katherine B Bateman, __Eisa Mahyari__, Archana Thomas, Stephanie L Junell, Theodore R Hobbs, Lauren D Martin, Rhonda MacAllister, Benjamin N Bimber, Mark K Slifka, Alfred W Legasse, Cassandra Moats, Michael K Axthelm, Jeremy Smedley, Anne D Lewis, Lois Colgin, Gabrielle Meyers, Richard T Maziarz, Benjamin J Burwitz, Jeffrey J Stanton, Jonah B Sacha. "Viral opportunistic infections in Mauritian cynomolgus macaques undergoing allogeneic stem cell transplantation mirror human transplant infectious disease complications." Xenotransplantation, 2020. [Link](https://doi.org/10.1111/xen.12578).

+ Emily B Wong, Marielle C Gold, Erin W Meermeier, Bongiwe Z Xulu, Sharon Khuzwayo, Zuri A Sullivan, __Eisa Mahyari__, Zoe Rogers, Hénrik Kløverpris, Prabhat K Sharma, Aneta H Worley, Umesh Lalloo, Prinita Baijnath, Anish Ambaram, Leon Naidoo, Moosa Suleman, Rajhmun Madansein, James E McLaren, Kristin Ladell, Kelly L Miners, David A Price, Samuel M Behar, Morten Nielsen, Victoria O Kasprowicz, Alasdair Leslie, William R Bishai, Thumbi Ndung’u, David M Lewinsohn. "TRAV1-2+ CD8+ T-cells including oligoconal expansions of MAIT cells are enriched in the airways in human tuberculosis." Nature Communications biology, 2019. [NCBI link](https://www.ncbi.nlm.nih.gov/pubmed/31231693/).

+ Yoshinori Fukazawa, Haesun Park, Mark J. Cameron, Francois Lefebvre, Richard Lum, Noel Coombes, __Eisa Mahyari__, Shoko I. Hagen, Jin Young Bae, Marcelo Delos Reyes, Tonya Swanson, Alfred W. Legasse, Andrew Sylwester, Scott G. Hansen, Andrew T. Smith, Petra Stafova, Rebecca Shoemaker, Yuan Li, Kelli Oswald, Michael K. Axthelm, Adrian McDermott, Guido Ferrari, David C. Montefiori, Paul T. Edlefsen, Michael Piatak, Jeffrey D. Lifson, Rafick P. Sékaly, and Louis J. Picker.   “Lymph node T cell responses predict the efficacy of live attenuated SIV vaccines.” Nature Medicine, 2012. [NCBI link](https://www.ncbi.nlm.nih.gov/pubmed/22961108).

## Accepted Abstracts, Poster, & Talks
+ Fall 2024     “Uncovering distinct and shared immunophenotypic signatures in ankylosing spondylitis (AS) and acute anterior uveitis (AAU): Applications of Machine-learning & AI in OMICs”, OHSU Dept. Medicine, Annual Research Retreat.
+ Summer 2023   “Uncovering the Biological Signal in Rhesus Macaque Testis: A Comparative Analysis of FFPE Probe- based and Frozen Poly-A- mRNA Spatial-omics Technologies”, European Testis Workshop Conference, Switzerland.
+ Spring 2022   “The Human Infertility Single-cell Testis Atlas (HISTA): An interactive molecular scRNA-seq reference of the human testis”, American Testis Workshop Conference, USA.
+ Fall 2021     "Comparative single-cell analysis of biopsies identifies pathogenic mechanisms in Klinefelter Syndrome (KS)", National Centers for Translational Research in Reproduction and Infertility (NCTRI) Conference.
+ Fall 2021     "Rhesus Immunome Reference Atlas (RIRA): A multi-tissue single-cell landscape of immune cells", 38th Symposium on NHP Models for AIDS Conference. 
+ Spring 2021   "Rhesus Immunome Reference Atlas (RIRA): An intro to the multi-tissue single-cell landscape of immune cells", OHSU-VGTI Weekly Seminar.
+ Spring 2019   "Identification of patient-specific molecular pathology from single-cell sequencing of human testis", American Society of Andrology (ASA) Annual Conference. 
+ Spring 2018   “Robust and reproducible classification of rare cellular subsets/signatures (RCS) in single-cell technologies within a transfer learning framework”, Research Week talk, OHSU.
+ Spring 2015   “Identification of novel MAIT subtypes with non-linear dimensionality reduction”, Research Week poster, OHSU.
+ Spring 2015   “A Reinforcement Learning Model for Solving the Protein Folding Problem”, BioDSP talk, OHSU
+ Winter 2014   “Latent Dirichlet Allocation and Applications in Biology”, Bioinformatics Discussion for Students and Postdocs (BioDSP) talk, OHSU.

## Teaching & Mentorship Experience
+ Summer 2022 Machine Learning and Single-cell OMICs analysis of with an international (France) master’s level graduate student. ONPRC.
+ Spring 2019 Bioinformatics & Computational Biology introductory training of a high-school student and a master’s level graduate student. ONPRC.
+ Spring 2017	Class TA & Mentored teaching, "Public and social-media engagement and the developing ‘avatar’ of an academic professional" for Research in Bioinformatics and Computational Biology with course professor: Eilis Boudreau M.D./Ph.D.
+ Spring 2016	Class TA and Mentored teaching, "Markov Models, Hidden Markov Models, and Markov Chain Mote Carlo Modeling" for Statistical methods with course professor: Shannon K. McWeeney Ph.D.
+ Spring 2015	Class TA for Statistical Methods with course professor Shannon K. McWeeney Ph.D.

## Software & Research Products

+ [“The Human Infertility Single-cell Testis Atlas (HISTA)”](https://conradlab.shinyapps.io/HISTA/): A web-based single-cell atlas that enables exploration and hypothesis testing of testis-specific cell types and genomic mechanisms in the context of infertility.

+ [“Spatial-Omics Testis Atlas (SPATA) Suite”](https://conradlab.shinyapps.io/SPATA/): A suite of tools to explore testis spatial and single-cell transcriptomic data across multiple species, facilitating comparative analysis and cross-species hypothesis generation in testis biology.

+ [“Mouse Spatial-Omics Testis Atlas (MoSpaTA)”](https://conradlab.shinyapps.io/MoSpaTA/): Includes spatial data from various technologies like StereoSeq, Curio, and 10X Genomics to provide a comprehensive view of mouse testis at different developmental stages.

+ [“Mouse Developmental Single-cell Testis Atlas (MoDSTA)”](https://conradlab.shinyapps.io/MoDSTA/): A curated atlas of mouse testis development from embryonic stages to adulthood, supporting developmental and reproductive research.

+ [“Rhesus macaque Spatial-Omics Testis Atlas (RheSpaTA)”](https://conradlab.shinyapps.io/RheSpaTA/): A platform housing spatial data from rhesus testis, enabling detailed investigation into non-human primate testis biology.

+ [“homo Sapian Spatial-Omics Testis Atlas (SapSpaTA)”](https://conradlab.shinyapps.io/SapSpaTA/): A spatial-omics dataset for human testis, offering insights into gene expression patterns and cell type-specific information related to human reproductive health.

+ [“ShinySDA R Package”](https://github.com/eisascience/ShinySDA): An R-based Shiny application for processing, browsing, and visualizing Sparse Decomposition of Arrays (SDA) objects, allowing real-time analysis of high-dimensional data.

+ [“Rhesus Immunome Reference Atlas (RIRA) R Package”](https://github.com/eisascience/RIRA): A multi-tissue single-cell RNA-seq atlas from rhesus macaque immune cells, complete with pipelines for cell type annotation and data integration for cross-species research.

+ [“Single-cell Custom Functions (scCustFx) R Package”](https://github.com/eisascience/scCustFx): A library of analytical and visualization functions for single-cell and spatial omics analysis, supporting custom workflows in R.

+ [“CellMembrane R Package”](https://github.com/eisascience/CellMembrane): A toolkit of high-level wrappers and visualization tools for single-cell omics data processing and analysis designed for fault-tolerant pipelines and enhanced QC visualizations.

+ [“TCR Enrichment Analysis (TEA)”](https://github.com/eisascience/Wong-Gold-Lewinsohn): A web-based visualization interface to evaluate differential enrichment of tissue-specific TCR clonotype.



## Awards and Honors
+ 2024 - 2026     NIH LRP - Contraception and Infertility through the National Institute of Child Health and Human Development (NICHD)
+ 2021 - 2023     NIH LRP - Contraception and Infertility through the National Institute of Child Health and Human Development (NICHD)
+ 2014 – 2016     Pulmonary/Critical Care Medicine. NIH T32 Grant, David Lewinsohn M.D./Ph.D., OHSU (5T32HL083808-06/07).
+ 2013 – 2014     Knight Cancer Institute, pre-doctoral funding, OHSU.
+ 2013 – 2014     Molecular Microbiology and Immunology, pre-doctoral funding, Paul Spellman Ph.D., OHSU.


## Professional Affiliations

- [International Society for Computational Biology (ISCB)](https://www.iscb.org)
- [American Association of Immunologists (AAI)](https://www.aai.org)
- [American Society of Andrology (ASA)](https://www.andrologysociety.org)

## Attended Professional Conferences

- 48th International Society of Andrology (ISA), Testis Workshop - Switzerland, 2023
- 47th International Society of Andrology (ISA), Testis Workshop - California, USA, 2022
- 39th Annual Symposium on Nonhuman Primate Models for AIDS - Oregon, USA, 2022
- CYTO 2016: Annual Congress of the International Society for Advancement of Cytometry - Seattle, WA, USA, 2016
- AAI Advanced Course in Immunology - Boston, MA, USA, 2014
- 29th Annual Symposium on Nonhuman Primate Models for AIDS - Seattle, WA, USA, 2011
- 28th Annual Symposium on Nonhuman Primate Models for AIDS - New Orleans, LA, USA, 2010



[See My LinkedIn](https://www.linkedin.com/in/eisa-mahyari-6605242a/)

[Follow @eisamahyari](https://twitter.com/eisamahyari)
