# Eisa Mahyari, Ph.D.

## Contact 
[Google Scholar Citations](https://scholar.google.com/citations?user=vp6qss8AAAAJ&hl=en), [Research Gate](https://www.researchgate.net/profile/Eisa_Mahyari), [ORCID](https://orcid.org/0000-0002-5613-3131), [Github](https://eisascience.github.io/), [Twitter](https://twitter.com/eisamahyari), [Email](mailto:eisamahyari@gmail.com)
 
## Education
__Ph.D.__ - Bioinformatics and Computational Biology, OHSU, Department of Medical Informatics and Clinical Epidemiology (DMICE), Division of Bioinformatics and Computational Biology (BCB), Portland, OR. “Robust and reproducible classification of rare cellular subsets/signatures (RCS) in single-cell technologies within a transfer learning framework.” [(doi:10.6083/NX9EZJ)](https://doi.org/10.6083/nx9ezj). Committee members: Shannon K. McWeeney Ph.D., David M. Lewinsohn MD./Ph.D., Michael Mooney Ph.D., Christina Zheng Ph.D., and Evan Lind Ph.D. (2013 – 2018). 

__M.S.__ - Biochemistry and Molecular Biology. Oregon Health and Science University (OHSU), Div. Environmental and Biomolecular Systems, Portland, OR. “A six-compartment Physiologically Based Pharmacokinetic (PBPK) Markov-chain Monte Carlo Model to simulate and quantify nicotine metabolism in human populations.” (2006 – 2008).

__B.S.__ - Biology and Chemistry double major. Portland State University (PSU), Portland, OR. (2002 – 2006)

## Research Statement
I am a computational biologist focused on multi-disciplinary translational biomedical research with expertise in machine learning, immunology, and male reproductive biology. 
My day-to-day tasks are mostly researched focused, writing code, running statistical tests, and translating the biology to meaninful and actionable targets. 
I also mentor and guide prospective or current graduate students both on their work and career development. I am fascinated by how the immune system, throughout life, functions and influences the development, health, and death of an organism. Specifically, leveraging modern technology, computational power, and statistics to bring about new insight and information, as well as testing research hypotheses that were not previously possible. 
Currently, I am focused on two domains of biomedical research split between the labs of Don F. Conrad Ph.D., and Benjamin Bimber Ph.D., and mainly work with single-cell transcriptomic and spatial-omics data. 
In the Conrad Lab, broadly, we investigate the genetics of infertility and in the Bimber lab, we are focused on the immune system, often, T cells, in the context of several infectious diseases. 

### Domains of research expertise/interest
- Reproductive Biology: Human, Rhesus, and Murine models. Specialized focus in testis biology and genetics of infertility. 
- Immunology: Human, Rhesus, and Murine models. Specialized focus in adaptive immune system and specifically T cell biology and their TCRs.
- Infectious Diseases: HIV/SIV, SARS-CoV2, ZIKA, Yellow Fever, Dengue, and Mycobacterium tuberculosis.
- Computational science: Statistical inference (Machine learning) with specialized focus in Matrix and Tensor decomposition methods and Transfer learning. 
- Mathematical modeling: with emphasis in Physiological Based Pharmacokinetic (PBPK) modeling with Markov Chain Monte-Carlo solutions of the dynamic models
- Statistical data-science: with emphasis on transcriptomic expression data and genomics variant data processing.
- Programming and scripting with R (dominantly), Python, and bash.
- High-performance cloud computing with SLURM.
- Project management and team leadership.


## Professional Experience
2018 - Current   __Computational Biologist, Molecular Immunopathology - RA__, Lab of Benjamin N. Bimber Ph.D. - Primate Genetics, ONPRC-OHSU.
- Generated hypotheses and translated research data to insights that improve the understanding of the immunopathology and identified genomic signatures of pathogen infection.
- Designed and implemented algorithms and statistical/machine learning models to drive discovery and interpretation using genomic, metagenomic, and other metadata; focused on single cell transcriptomics.
- Managed and consulted on collaborations and projects
- Provided technical (computational, statistical, programming, etc) training to researchers and students

2018 - Current   __Computational Biologist, Reproductive biology - RA__, Lab of Donald F. Conrad Ph.D. - Reproductive Genetics, ONPRC-OHSU.
- Identified pathological genomic signatures that distinguished clinical differences in male infertility.
- Designed scRNASeq experiments to address specific hypotheses with mouse models.
- Implemented and modified per need, current and emerging computational biology and bioinformatics algorithms
- Collaborated with other scientists, physicians, and engineers focused on scientific discoveries and the invention of technologies.

2016 - 2018      __Doctoral Candidate__, DMICE-BCB, OHSU. Developed novel machine learning framework for automated classification of rare cells in single-cell RNA-seq data.
2013 - 2016      __Doctoral Student__, DMICE-BCB, OHSU. 
Research Involvement:
__Genome-based Bioinformatics__, Assessment of bioinformatics pipelines identifying genomic structural variants across several sequenced Gibbons. Lucia Carbone Ph.D.
__Computational phenotyping in cancer cell lines__, CyTOF-based computational analysis of phenotypic heterogeneity in breast cancer-derived cell in the context of combinatory anti-cancer drug treatments to identify sensitive phenotypic subtypes within several breast cancer cell lines. Paul T. Spellman Ph.D.
__Computational immunophenotyping__, CyTOF-based computational immunophenotypic analysis of a unique population of tuberculosis (TB)-recognizing T-cells known as mucosal associated invariant T (MAIT) cells. David M. Lewinsohn M.D./Ph.D.

2008 - 2012      __Sr. Research Associate__, Louis Picker M.D., Vaccine & Gene Therapy Institute (VGTI), OHSU. 
Contribution: Performed experiments, analyzed flow cytometric data, performed statistical tests, and developed reports and presentation martials. 

2005 - 2007      __Research Associate__, Research Associate, Clinical Research Investigative Studies Program (CRISP) and Critical Care Academics Associates Program (CCAAP), OHSU.
Contribution: Acquired clinical data from patient charts for ongoing studies and identified candidates for enrollment into qualifying research in the emergency department and the intensive care units.  



## Publications

+ __Eisa Mahyari__, Jingtao Guo, Ana C. Lima, Daniel P. Lewinsohn, Alexandra M. Stendahl, Katinka A. Vigh-Conrad, Xichen Nie, Liina Nagirnaja, Nicole B. Rockweiler, Douglas T.Carrell, James M.Hotaling, Kenneth I.Aston, Donald F.Conrad  "Comparative single-cell analysis of biopsies clarifies pathogenic mechanisms in Klinefelter syndrome." The American Journal of Human Genetics 108, no. 10 (2021): 1924-1945. [link](https://www.sciencedirect.com/science/article/abs/pii/S0002929721003372)

+ Bruce K. Patterson, Harish Seethamraju, Kush Dhody, Michael J. Corley, Kazem Kazempour, Jay Lalezari, Alina PS Pang, __Eisa Mahyari__ et al. "CCR5 inhibition in critical COVID-19 patients decreases inflammatory cytokines, increases CD8 T-cells, and decreases SARS-CoV2 RNA in plasma by day 14." International Journal of Infectious Diseases 103 (2021): 25-32.[Link](https://www.sciencedirect.com/science/article/pii/S1201971220323055)

+ Husam Taher, __Eisa Mahyari__, Craig Kreklywich, Luke S Uebelhoer, Matthew R McArdle, Matilda J Moström, Amruta Bhusari, Michael Nekorchuk, Travis Whitmer, Elizabeth A Scheef, Lesli M Sprehe, Dawn Roberts, Colette M Hughes, Kerianne A Jackson, Andrea N Selseth, Abigail B Ventura, Yujuan Yue, Kimberli A Schmidt, Jason Shao, Paul T Edlefsen, Jeremy Smedley, Richard J Stanton, Michael K Axthelm, Jacob D Estes, Scott G Hansen, Amitinder Kaur, Peter A Barry, Benjamin N Bimber, Louis J Picker, Daniel N Streblow, Klaus Früh, Daniel Malouli. "In vitro and in vivo characterization of a recombinant rhesus cytomegalovirus containing a complete genome." bioRxiv, 2020. [Link](https://doi.org/10.1101/2020.06.02.129486).

+ Helen L Wu, Whitney C Weber, Christine Shriver‐Munsch, Tonya Swanson, Mina Northrup, Heidi Price, Kimberly Armantrout, Mitchell Robertson‐LeVay, Jason S Reed, Katherine B Bateman, __Eisa Mahyari__, Archana Thomas, Stephanie L Junell, Theodore R Hobbs, Lauren D Martin, Rhonda MacAllister, Benjamin N Bimber, Mark K Slifka, Alfred W Legasse, Cassandra Moats, Michael K Axthelm, Jeremy Smedley, Anne D Lewis, Lois Colgin, Gabrielle Meyers, Richard T Maziarz, Benjamin J Burwitz, Jeffrey J Stanton, Jonah B Sacha. "Viral opportunistic infections in Mauritian cynomolgus macaques undergoing allogeneic stem cell transplantation mirror human transplant infectious disease complications." Xenotransplantation, 2020. [Link](https://doi.org/10.1111/xen.12578).

+ Emily B Wong, Marielle C Gold, Erin W Meermeier, Bongiwe Z Xulu, Sharon Khuzwayo, Zuri A Sullivan, __Eisa Mahyari__, Zoe Rogers, Hénrik Kløverpris, Prabhat K Sharma, Aneta H Worley, Umesh Lalloo, Prinita Baijnath, Anish Ambaram, Leon Naidoo, Moosa Suleman, Rajhmun Madansein, James E McLaren, Kristin Ladell, Kelly L Miners, David A Price, Samuel M Behar, Morten Nielsen, Victoria O Kasprowicz, Alasdair Leslie, William R Bishai, Thumbi Ndung’u, David M Lewinsohn. "TRAV1-2+ CD8+ T-cells including oligoconal expansions of MAIT cells are enriched in the airways in human tuberculosis." Nature Communications biology, 2019. [NCBI link](https://www.ncbi.nlm.nih.gov/pubmed/31231693/).

+ Yoshinori Fukazawa, Haesun Park, Mark J. Cameron, Francois Lefebvre, Richard Lum, Noel Coombes, __Eisa Mahyari__, Shoko I. Hagen, Jin Young Bae, Marcelo Delos Reyes, Tonya Swanson, Alfred W. Legasse, Andrew Sylwester, Scott G. Hansen, Andrew T. Smith, Petra Stafova, Rebecca Shoemaker, Yuan Li, Kelli Oswald, Michael K. Axthelm, Adrian McDermott, Guido Ferrari, David C. Montefiori, Paul T. Edlefsen, Michael Piatak, Jeffrey D. Lifson, Rafick P. Sékaly, and Louis J. Picker.   “Lymph node T cell responses predict the efficacy of live attenuated SIV vaccines.” Nature Medicine, 2012. [NCBI link](https://www.ncbi.nlm.nih.gov/pubmed/22961108).

## Accepted Abstracts, Poster, & Talks
+ Fall 2021     "Comparative single-cell analysis of biopsies identifies pathogenic mechanisms in Klinefelter Syndrome (KS)", National Centers for Translational Research in Reproduction and Infertility (NCTRI) Conference.
+ Fall 2021     "Rhesus Immunome Reference Atlas (RIRA): A multi-tissue single-cell landscape of immune cells", 38th Symposium on NHP Models for AIDS Conference. 
+ Spring 2021   "Rhesus Immunome Reference Atlas (RIRA): An intro to the multi-tissue single-cell landscape of immune cells", OHSU-VGTI Weekly Seminar.
+ Spring 2019   "Identification of patient-specific molecular pathology from single-cell sequencing of human testis", American Society of Andrology (ASA) Annual Conference. 
+ Spring 2018   “Robust and reproducible classification of rare cellular subsets/signatures (RCS) in single-cell technologies within a transfer learning framework”, Research Week talk, OHSU.
+ Spring 2015   “Identification of novel MAIT subtypes with non-linear dimensionality reduction”, Research Week poster, OHSU.
+ Spring 2015   “A Reinforcement Learning Model for Solving the Protein Folding Problem”, BioDSP talk, OHSU
+ Winter 2014   “Latent Dirichlet Allocation and Applications in Biology”, Bioinformatics Discussion for Students and Postdocs (BioDSP) talk, OHSU.

## Teaching Experience
+ Spring 2017	Class TA & Mentored teaching, "Public and social-media engagement and the developing ‘avatar’ of an academic professional" for Research in Bioinformatics and Computational Biology with course professor: Eilis Boudreau M.D./Ph.D.
+ Spring 2016	Class TA and Mentored teaching, "Markov Models, Hidden Markov Models, and Markov Chain Mote Carlo Modeling" for Statistical methods with course professor: Shannon K. McWeeney Ph.D.
+ Spring 2015	Class TA for Statistical methods with course professor: Shannon K. McWeeney Ph.D.

## MCL Research Products
+ [“The Human Infertility Single-cell Testis Atlas (HISTA)”](https://conradlab.shinyapps.io/HISTA/). A web-based visualization interface to evaluate differential enrichment of tissue-specific TCR clonotype. 
+ [“TCR Enrichment Analysis (TEA)”](https://github.com/eisascience/Wong-Gold-Lewinsohn). A web-based visualization interface to evaluate differential enrichment of tissue-specific TCR clonotype. 


## Awards and Honors
+ 2021 - 2022     NIH LRP - Contraception and Infertility Research.
+ 2014 – 2016     Pulmonary/Critical Care Medicine. NIH T32 Grant, David Lewinsohn M.D./Ph.D., OHSU (5T32HL083808-06/07).
+ 2013 – 2014     Knight Cancer Institute, pre-doctoral funding, OHSU.
+ 2013 – 2014     Molecular Microbiology and Immunology, pre-doctoral funding, Paul Spellman Ph.D., OHSU.

## Associations and Memberships
+ Chair, Communications subcommittee, VGTI Diversity Equity and Inclusion (DEI) Committee
+ Member, VGTI Scientific Retreat committee

[Follow @eisamahyari](https://twitter.com/eisamahyari)
