# Eisa Mahyari, Ph.D.

## Contact 
[Google Scholar Citations](https://scholar.google.com/citations?user=vp6qss8AAAAJ&hl=en) | [Research Gate](https://www.researchgate.net/profile/Eisa_Mahyari) | [ORCID](https://orcid.org/0000-0002-5613-3131) | [GitHub](https://eisascience.github.io/) | [Twitter](https://twitter.com/eisamahyari) | [Email](mailto:eisamahyari@gmail.com)

## Education
- **Post-Ph.D.** – Mammalian Reproduction, Oregon Health & Science University (OHSU), Oregon National Primate Research Center (ONPRC), Division of Genetics  
  *Advisor: Prof. Don F. Conrad*
  
- **Ph.D.** – Bioinformatics and Computational Biology, OHSU, Department of Medical Informatics and Clinical Epidemiology (DMICE), Division of Bioinformatics and Computational Biology (BCB), Portland, OR  
  *Dissertation*: "Robust and reproducible classification of rare cellular subsets/signatures (RCS) in single-cell technologies within a transfer learning framework." [(doi:10.6083/NX9EZJ)](https://doi.org/10.6083/nx9ezj)  
  *Committee*: Shannon K. McWeeney, Ph.D., David M. Lewinsohn, M.D./Ph.D., Michael Mooney, Ph.D., Christina Zheng, Ph.D., Evan Lind, Ph.D.  
  *(2013 – 2018)*

- **M.S.** – Biochemistry and Molecular Biology, OHSU, Division of Environmental and Biomolecular Systems, Portland, OR  
  *Thesis*: "A six-compartment Physiologically Based Pharmacokinetic (PBPK) Markov-chain Monte Carlo Model to simulate and quantify nicotine metabolism in human populations."  
  *(2006 – 2008)*

- **B.S.** – Biology and Chemistry, Portland State University (PSU), Portland, OR  
  *(2002 – 2006)*

## Research Statement
I am a computational biologist dedicated to translational biomedical research, integrating machine learning, immunology, and male reproductive biology to drive scientific innovation. My expertise includes leading diverse projects, coding, performing statistical analyses, and converting complex biological data into actionable insights alongside principal investigators and cross-disciplinary teams.

In my current role in Dr. Don F. Conrad’s lab at OHSU, I investigate the genomics of reproduction and the genetics of infertility, primarily focusing on testis biology and multi-omics data. This aligns with my broader research interests, emphasizing integrating high-dimensional multi-omics datasets, performing cross-species analysis, and employing statistical inference and computational omics to support clinical and preclinical initiatives.

I actively consult across labs, providing technical training and mentorship to graduate students guiding their research and career development. I am passionate about exploring the immune system's role in organismal health, development, and aging. By harnessing advanced computational methods, I aim to address research questions that were previously inaccessible.

My career has been defined by consistent advancement and a focus on statistical, machine learning, and data analytic methods relevant to translational and clinical research. I have played a pivotal role in designing, implementing, and analyzing extensive research projects, combining wet lab and in silico methodologies to develop hypotheses and transform data into actionable outcomes. My work upholds rigorous quality control across various omics data types to ensure integrity and reproducibility. Through statistical consultations and analyses, I have contributed significantly to clinical study documentation, scientific publications, presentations, grants, and regulatory filings in both academic and industry settings.

Currently, I am interested in leading and developing teams to create software, pipelines, and testing methodologies tailored for analyzing and translating complex biomedical data, supporting impactful translational research by integrating data types and applying robust statistical methodologies and computational tools.

### Domains of Research Expertise & Interest
- **Reproductive Biology**: Testis biology and genetics of infertility in human, rhesus, and murine models.
- **Immunology**: Focused on adaptive immunity, with an emphasis on T cell biology and T cell receptors (TCRs) across human, rhesus, and murine models.
- **Infectious Diseases**: Research in HIV/SIV, SARS-CoV-2, ZIKA, Yellow Fever, Dengue, and Mycobacterium tuberculosis.
- **Computational Science**: Statistical inference and machine learning, specializing in matrix/tensor decomposition and transfer learning.
- **Large Language Models (LLMs)**: Applications and methodologies for LLMs in biomedical research.
- **Mathematical Modeling**: Emphasis on Physiological Based Pharmacokinetic (PBPK) modeling, with Markov Chain Monte Carlo solutions for dynamic models.
- **Statistical Data Science**: Expertise in processing transcriptomic expression data and genomic variant data.
- **Programming and Scripting**: Advanced proficiency in R (primary), Python, and Bash.
- **High-Performance Cloud Computing**: Experience with SLURM for scalable computational tasks.
- **Project Management & Team Leadership**: Leading cross-functional teams and managing complex, multi-disciplinary projects.


## Professional Experience
2022 - Current   __Research Assistant Professor, Mammalian Reproduction__, Lab of Donald F. Conrad Ph.D. - Div. Genetics, ONPRC-OHSU.

2018 - 2022   __Computational Biologist, Molecular Immunopathology - RA__, Lab of Benjamin N. Bimber Ph.D. - Primate Genetics, ONPRC-OHSU.
- Generated hypotheses and translated research data to insights that improve the understanding of the immunopathology and identified genomic signatures of pathogen infection.
- Designed and implemented algorithms and statistical/machine learning models to drive discovery and interpretation using genomic, metagenomic, and other metadata; focused on single cell transcriptomics.
- Managed and consulted on collaborations and projects
- Provided technical (computational, statistical, programming, etc) training to researchers and students

2022 - 2022   __Biostatistics & Computational Biology Consultant.__, Drug development & testing for nonalcoholic steatohepatitis (NASH) -  CytoDyn, USA. 
- Designed and implemented analysis frameworks for multi-omics data, integrating statistical and machine learning approaches to support therapeutic hypotheses for NASH.
- Conducted extensive quality control across clinical data, directly supporting clinical-stage program milestones.
- Collaborated with cross-disciplinary teams to translate complex results into actionable insights, supporting drug treatment decisions for clinical trials.

2018 - 2022   __Computational Biologist, Reproductive biology - RA__, Lab of Donald F. Conrad Ph.D. - Reproductive Genetics, ONPRC-OHSU.
- Identified pathological genomic signatures that distinguished clinical differences in male infertility.
- Designed scRNASeq experiments to address specific hypotheses with mouse models.
- Implemented and modified per need, current and emerging computational biology and bioinformatics algorithms
- Collaborated with other scientists, physicians, and engineers focused on scientific discoveries and the invention of technologies.

2016 - 2018      __Doctoral Candidate__, DMICE-BCB, OHSU. Developed novel machine learning framework for automated classification of rare cells in single-cell RNA-seq data.
2013 - 2016      __Doctoral Student__, DMICE-BCB, OHSU. 
Research Involvement:
__Genome-based Bioinformatics__, Assessment of bioinformatics pipelines identifying genomic structural variants across several sequenced Gibbons. Lucia Carbone Ph.D.
__Computational phenotyping in cancer cell lines__, CyTOF-based computational analysis of phenotypic heterogeneity in breast cancer-derived cell in the context of combinatory anti-cancer drug treatments to identify sensitive phenotypic subtypes within several breast cancer cell lines. Paul T. Spellman Ph.D.
__Computational immunophenotyping__, CyTOF-based computational immunophenotypic analysis of a unique population of tuberculosis (TB)-recognizing T-cells known as mucosal associated invariant T (MAIT) cells. David M. Lewinsohn M.D./Ph.D.

2008 - 2012      __Sr. Research Associate__, Louis Picker M.D., Vaccine & Gene Therapy Institute (VGTI), OHSU. 
Contribution: Performed experiments, analyzed flow cytometric data, performed statistical tests, and developed reports and presentation martials. 

2005 - 2007      __Research Associate__, Research Associate, Clinical Research Investigative Studies Program (CRISP) and Critical Care Academics Associates Program (CCAAP), OHSU.
Contribution: Acquired clinical data from patient charts for ongoing studies and identified candidates for enrollment into qualifying research in the emergency department and the intensive care units.  



## Publications by year

+ Liina Nagirnaja, Alexandra M. Lopes, Wu-Lin Charng, Brian Miller, Rytis Stakaitis, Ieva Golubickaite, Alexandra Stendahl, Tianpengcheng Luan, Corinna Friedrich, __Eisa Mahyari__, [...], Kristian Almstrup, Kenneth I Aston, Donald F Conrad "Diverse monogenic subforms of human spermatogenic failure", Nature Communications, 13, 1, (2022). [link](https://doi.org/10.1038/s41467-022-35661-z)

+ Yoshinori Fukazawa, Jeremy Smedley, Bryan Randall, __Eisa Mahyari__, Carly Starke, Richard Lum, Manuel Medina, Benjamin Varco-Merth, Kathleen Busman-Sahay, Miranda Fisher, "Mesenteric lymph nodes are sites of increased SIV RNA expression in SIV-infected rhesus macaques on antiretroviral therapy", Journal of Medical Primatology, 51, 5, 320-320, (2022)

+ GW McElfresh, Gregory Boggy, __Eisa Mahyari__, Michael Stutz, Abigail Ventura, Scott Hansen, Louis Picker, Benjamin Bimber, "Single cell transcriptomic profiling of early tuberculosis infection and granuloma formation in rhesus macaque", Journal of Medical Primatology, 51, 5, 313-314, (2022)

+ __Eisa Mahyari__, Abigail Ventura, Eric McDonald, Gregory Boggy, GW McElfresh, Scott Hansen, Louis Picker, Benjamin Bimber, "Rhesus Immunome Reference Atlas (RIRA): A multi-tissue single-cell landscape of immune cells", Journal of Medical Primatology, 51, 5, 296-296, (2022)

+ Gregory J Boggy, GW McElfresh, __Eisa Mahyari__, Abigail B Ventura, Scott G Hansen, Louis J Picker, Benjamin N Bimber "BFF and cellhashR: analysis tools for accurate demultiplexing of cell hashing data." Bioinformatics (2022). [link](https://academic.oup.com/bioinformatics/article-abstract/38/10/2791/6585402)

+ __Eisa Mahyari__, Jingtao Guo, Ana C. Lima, Daniel P. Lewinsohn, Alexandra M. Stendahl, Katinka A. Vigh-Conrad, Xichen Nie, Liina Nagirnaja, Nicole B. Rockweiler, Douglas T.Carrell, James M.Hotaling, Kenneth I.Aston, Donald F.Conrad  "Comparative single-cell analysis of biopsies clarifies pathogenic mechanisms in Klinefelter syndrome." The American Journal of Human Genetics 108, no. 10 (2021): 1924-1945. [link](https://www.sciencedirect.com/science/article/abs/pii/S0002929721003372)

+ N Haigwood, M Shapiro, T Cheever, S Pandey, __E Mahyari__, K Onwuzu, J Reed, H Sidener, J Smedley, L Colgin, A Lewis, A Johnson, B Bimber, J Sacha, A Hessell "Immune perturbation is more profound in newborn than in infant macaques during acute SHIV infection." Journal of the International AIDS Society (2021). [link](https://go.gale.com/ps/i.do?id=GALE%7CA656303588&sid=googleScholar&v=2.1&it=r&linkaccess=abs&issn=17582652&p=AONE&sw=w&userGroupName=oregon_oweb)

+ Bruce K. Patterson, Harish Seethamraju, Kush Dhody, Michael J. Corley, Kazem Kazempour, Jay Lalezari, Alina PS Pang, __Eisa Mahyari__ et al. "CCR5 inhibition in critical COVID-19 patients decreases inflammatory cytokines, increases CD8 T-cells, and decreases SARS-CoV2 RNA in plasma by day 14." International Journal of Infectious Diseases 103 (2021): 25-32.[Link](https://www.sciencedirect.com/science/article/pii/S1201971220323055)

+ Husam Taher, __Eisa Mahyari__, Craig Kreklywich, Luke S Uebelhoer, Matthew R McArdle, Matilda J Moström, Amruta Bhusari, Michael Nekorchuk, Travis Whitmer, Elizabeth A Scheef, Lesli M Sprehe, Dawn Roberts, Colette M Hughes, Kerianne A Jackson, Andrea N Selseth, Abigail B Ventura, Yujuan Yue, Kimberli A Schmidt, Jason Shao, Paul T Edlefsen, Jeremy Smedley, Richard J Stanton, Michael K Axthelm, Jacob D Estes, Scott G Hansen, Amitinder Kaur, Peter A Barry, Benjamin N Bimber, Louis J Picker, Daniel N Streblow, Klaus Früh, Daniel Malouli. "In vitro and in vivo characterization of a recombinant rhesus cytomegalovirus containing a complete genome." PLoS pathogens, 2020. [Link](https://doi.org/10.1371/journal.ppat.1008666).

+ Helen L Wu, Whitney C Weber, Christine Shriver‐Munsch, Tonya Swanson, Mina Northrup, Heidi Price, Kimberly Armantrout, Mitchell Robertson‐LeVay, Jason S Reed, Katherine B Bateman, __Eisa Mahyari__, Archana Thomas, Stephanie L Junell, Theodore R Hobbs, Lauren D Martin, Rhonda MacAllister, Benjamin N Bimber, Mark K Slifka, Alfred W Legasse, Cassandra Moats, Michael K Axthelm, Jeremy Smedley, Anne D Lewis, Lois Colgin, Gabrielle Meyers, Richard T Maziarz, Benjamin J Burwitz, Jeffrey J Stanton, Jonah B Sacha. "Viral opportunistic infections in Mauritian cynomolgus macaques undergoing allogeneic stem cell transplantation mirror human transplant infectious disease complications." Xenotransplantation, 2020. [Link](https://doi.org/10.1111/xen.12578).

+ Emily B Wong, Marielle C Gold, Erin W Meermeier, Bongiwe Z Xulu, Sharon Khuzwayo, Zuri A Sullivan, __Eisa Mahyari__, Zoe Rogers, Hénrik Kløverpris, Prabhat K Sharma, Aneta H Worley, Umesh Lalloo, Prinita Baijnath, Anish Ambaram, Leon Naidoo, Moosa Suleman, Rajhmun Madansein, James E McLaren, Kristin Ladell, Kelly L Miners, David A Price, Samuel M Behar, Morten Nielsen, Victoria O Kasprowicz, Alasdair Leslie, William R Bishai, Thumbi Ndung’u, David M Lewinsohn. "TRAV1-2+ CD8+ T-cells including oligoconal expansions of MAIT cells are enriched in the airways in human tuberculosis." Nature Communications biology, 2019. [NCBI link](https://www.ncbi.nlm.nih.gov/pubmed/31231693/).

+ Yoshinori Fukazawa, Haesun Park, Mark J. Cameron, Francois Lefebvre, Richard Lum, Noel Coombes, __Eisa Mahyari__, Shoko I. Hagen, Jin Young Bae, Marcelo Delos Reyes, Tonya Swanson, Alfred W. Legasse, Andrew Sylwester, Scott G. Hansen, Andrew T. Smith, Petra Stafova, Rebecca Shoemaker, Yuan Li, Kelli Oswald, Michael K. Axthelm, Adrian McDermott, Guido Ferrari, David C. Montefiori, Paul T. Edlefsen, Michael Piatak, Jeffrey D. Lifson, Rafick P. Sékaly, and Louis J. Picker.   “Lymph node T cell responses predict the efficacy of live attenuated SIV vaccines.” Nature Medicine, 2012. [NCBI link](https://www.ncbi.nlm.nih.gov/pubmed/22961108).

## Accepted Abstracts, Poster, & Talks
+ Spring 2021   "The human infertility scRNA-seq testis atlas (HISTA)", Testis workshop 2021.
+ Fall 2021     "Comparative single-cell analysis of biopsies identifies pathogenic mechanisms in Klinefelter Syndrome (KS)", National Centers for Translational Research in Reproduction and Infertility (NCTRI) Conference.
+ Fall 2021     "Rhesus Immunome Reference Atlas (RIRA): A multi-tissue single-cell landscape of immune cells", 38th Symposium on NHP Models for AIDS Conference. 
+ Spring 2021   "Rhesus Immunome Reference Atlas (RIRA): An intro to the multi-tissue single-cell landscape of immune cells", OHSU-VGTI Weekly Seminar.
+ Spring 2019   "Identification of patient-specific molecular pathology from single-cell sequencing of human testis", American Society of Andrology (ASA) Annual Conference. 
+ Spring 2018   “Robust and reproducible classification of rare cellular subsets/signatures (RCS) in single-cell technologies within a transfer learning framework”, Research Week talk, OHSU.
+ Spring 2015   “Identification of novel MAIT subtypes with non-linear dimensionality reduction”, Research Week poster, OHSU.
+ Spring 2015   “A Reinforcement Learning Model for Solving the Protein Folding Problem”, BioDSP talk, OHSU
+ Winter 2014   “Latent Dirichlet Allocation and Applications in Biology”, Bioinformatics Discussion for Students and Postdocs (BioDSP) talk, OHSU.

## Teaching Experience
+ Spring 2017	Class TA & Mentored teaching, "Public and social-media engagement and the developing ‘avatar’ of an academic professional" for Research in Bioinformatics and Computational Biology with course professor: Eilis Boudreau M.D./Ph.D.
+ Spring 2016	Class TA and Mentored teaching, "Markov Models, Hidden Markov Models, and Markov Chain Mote Carlo Modeling" for Statistical methods with course professor: Shannon K. McWeeney Ph.D.
+ Spring 2015	Class TA for Statistical methods with course professor: Shannon K. McWeeney Ph.D.

## MCL Research Products
+ [“The Human Infertility Single-cell Testis Atlas (HISTA)”](https://conradlab.shinyapps.io/HISTA/). A web-based single-cell atlas that enables exploration and hypothesis testing of testis-specific cell types and genomic mechanisms in the context of infertility.
+ [“TCR Enrichment Analysis (TEA)”](https://github.com/eisascience/Wong-Gold-Lewinsohn). A web-based visualization interface to evaluate differential enrichment of tissue-specific TCR clonotype. 


## Awards and Honors
+ 2021 - 2022     NIH LRP - Contraception and Infertility Research.
+ 2014 – 2016     Pulmonary/Critical Care Medicine. NIH T32 Grant, David Lewinsohn M.D./Ph.D., OHSU (5T32HL083808-06/07).
+ 2013 – 2014     Knight Cancer Institute, pre-doctoral funding, OHSU.
+ 2013 – 2014     Molecular Microbiology and Immunology, pre-doctoral funding, Paul Spellman Ph.D., OHSU.

## Associations and Memberships
+ Chair, Communications subcommittee, VGTI Diversity Equity and Inclusion (DEI) Committee
+ Member, VGTI Scientific Retreat committee

[Follow @eisamahyari](https://twitter.com/eisamahyari)
